frontend/packages/dev-console/src/components/helm/HelmInstallUpgradePage.tsx

```diff
@@ -160,12 +164,45 @@ const HelmInstallUpgradePage: React.FunctionComponent<HelmInstallUpgradePageProp
     return <LoadingBox />;
   }
 
+  const handleChartVersionChange = (chart: HelmChart) => {
+    setChartReadme(getHelmChartReadme(chart));
+  };
+
+  const launchReadmeModal = () => {
+    import('./HelmReadmeModal')
+      .then((m) =>
+        m.helmReadmeModalLauncher({
+          readme: chartReadme,
+          modalClassName: 'odc__modal-xl',
```

Comment:
Better to use the global class and size that is being used everywhere else in console. Then you won't need any special CSS.

```suggestion
          modalClassName: 'modal-lg',
```

Reply:
updated to use large modal(modal-lg) instead of extra large modal (modal-xl)